docs/how-to-guides/integrating-autoware/creating-maps/creating-vector-map/index.md

```diff
@@ -0,0 +1,26 @@
+# Creating a vector map
+
+## Overview
+
+In this section, we will explain how to create Lanelet2 maps with TIER IV's [vector map builder tool](https://tools.tier4.jp/feature/vector_map_builder_ll2/).
+
+If you want to look at another method,
+[MapToolbox](https://github.com/autocore-ai/MapToolbox) is a plugin for [Unity](https://unity.com/) specifically designed to create Lanelet2 maps for Autoware.
+We didn't recommend [JOSM](https://josm.openstreetmap.de/) since it needs modifications for Autoware usage.
+So, this process can be tedious and time-consuming.
```

Comment:
Technically speaking, the other tools should still be able to create maps for Autoware without modifications, but TIER IV's tool has a presets for easier creation of Autoware Lanelet2 Map.

I would phrase it like the following: 
```suggestion
There are alternative tools such as
Unity-based app [MapToolbox](https://github.com/autocore-ai/MapToolbox) and 
Java-based app [JOSM](https://josm.openstreetmap.de/) that you may use for creating Lanelet2 map.
We will be using TIER IV's Vector Map Builder in the tutorial since it works on a browser without installation of extra dependency applications.  
```

Reply:
Done :+1: